src/lib/chips/chips.scss

```diff
@@ -150,6 +146,17 @@ $mat-chip-remove-size: 18px;
   }
 }
 
+.mat-standard-chip.mat-chip-disabled {
```

Comment:
This can be moved up into the `.mat-standard-chip` styles so all the standard chip styling is in the same place.